home-manager/desktop.nix

```diff
@@ -46,6 +48,11 @@ in
 {
   imports = [ ./gnome.nix ];
 
+  packages = [
+    pkgs.unstable.kanata # Don't require kanata-with-cmd for now
+    inputs.kanata-tray.packages.${hostPlatform.system}.kanata-tray
+  ];
```

Comment:
Put at NixOS layer might better, even through the kanata-tray is only required for users